Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "Syslogs"
+uuid = "cea106d9-e007-5e6c-ad93-58fe2094e9c4"
+authors = ["Rory Finnegan <rory.finnegan@gmail.com>"]
```

Comment:
Should be "Invenia Technical Computing Corporation"

Reply:
Just to check: have we documented this somewhere?